R/utils_render_latex.R

```diff
@@ -210,33 +223,39 @@ create_body_component_l <- function(data) {
         group_label = gsub("^NA", "\\textemdash", group_label))
   }
 
-  group_rows <- create_group_rows(n_rows, groups_rows_df, context = "latex")
-
-  if (stub_available && "__GT_ROWNAME_PRIVATE__" %in% names(body)) {
-    visible_vars <- c("__GT_ROWNAME_PRIVATE__", default_vars)
-  } else if (stub_available) {
-    visible_vars <- c(dt_boxhead_get_var_stub(data), default_vars)
-  } else {
-    visible_vars <- default_vars
-  }
-
-  # Split `body_content` by slices of rows and create data rows
-  body_content <- as.vector(t(body[, visible_vars]))
-  row_splits <- split(body_content, ceiling(seq_along(body_content) / n_cols))
-  data_rows <- create_data_rows(n_rows, row_splits, context = "latex")
+  group_rows <-
+    create_group_rows(
+      n_rows = n_rows,
+      groups_rows_df = groups_rows_df
+    )
 
   summary_rows <-
     create_summary_rows(
```

Comment:
If `create_summary_rows` is LaTeX-specific, let's rename to `create_summary_rows_l` or whatever